src/ng/components/selection/selection.core.js

```diff
@@ -36,6 +32,31 @@ class SelectionCore extends Directive(SELECTION_CORE_NAME, {view: `^^${VIEW_CORE
 		});
 	}
 
+	mode(mode) {
+		const currentMode = this.view.model.selection().mode;
+
+		if (!mode) {
+			return currentMode; 
+		}
+
+		return mode === currentMode;
+	}
+
+	get checkbox() {
+		return  this.view.model.selection().checkbox;
+	}
+
+	get items() {
+		switch (this.mode()) {
+			case 'row':
```

Comment:
also can we move selection.core to view.selection
and in template use $view.selection.toggle...
see. core/view